packages/react-composites/src/composites/CallComposite/MockCallAdapter.ts

```diff
@@ -136,6 +140,35 @@ export class MockCallAdapter implements CallAdapter {
   getEnvironmentInfo(): Promise<EnvironmentInfo> {
     throw Error('getEnvironmentInfo not implemented');
   }
+  /* @conditional-compile-remove(video-background-effects) */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  blurVideoBackground(bgBlurConfig?: BackgroundBlurConfig): Promise<void> {
+    throw new Error('blurVideoBackground not implemented.');
+  }
+  /* @conditional-compile-remove(video-background-effects) */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Avoid use // eslint-disable-next-line as much as possible, for here, you can use "_" as variable name, then you should be able to bypass the check